tests/profile/ProfileUiTest.php

```diff
@@ -1,13 +1,12 @@
 <?php
 
-class ProfileUiTest extends PHPUnit_Framework_TestCase
+require_once('AbstractProfileTest.php');
```

Comment:
Since you've got composer in this project now you could use its autoload and namespace everything then you won't need the `require_once`.